webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -0,0 +1,171 @@
+  };
+  return (
+    <>
+      <PageSection className="header" variant={PageSectionVariants.light}>
+        <div style={{ marginLeft: '18px', marginRight: '18px' }}>
+          <Breadcrumb style={{ marginTop: '15px' }}>
+            <BreadcrumbItem to="/hosts">Hosts</BreadcrumbItem>
+            <BreadcrumbItem isActive>
+              {response.name || <Skeleton />}
+            </BreadcrumbItem>
+          </Breadcrumb>
+          {/* TODO: Replace all br with css */}
+          <br />
+          <br />
+          <Grid>
+            <GridItem span={2}>
+              <Title headingLevel="h5" size="2xl">
+                {/* TODO: Make a generic Skeleton HOC (withSkeleton) */}
+                {response.name || <Skeleton />}
+              </Title>
+            </GridItem>
+            <GridItem style={{ marginTop: '5px', marginLeft: '10px' }} span={8}>
+              <Badge key={1}>{response.operatingsystem_name}</Badge>{' '}
+              <Badge key={21}>{response.architecture_name}</Badge>
+            </GridItem>
+            <GridItem span={2}>
+              <ActionsBar hostName={response.name || <Skeleton />} />
+            </GridItem>
+          </Grid>
+          <Text style={{ fontStyle: 'italic' }} component={TextVariants.p}>
+            {/* TODO: extracting text and remove timeago usage in favor i18n */}
+            {response.name ? (
+              <div>
+                created{' '}
+                <RelativeDateTime
+                  date={response.created_at}
+                  defaultValue="N/A"
+                />{' '}
+                by {response.owner_name} (updated{' '}
+                <RelativeDateTime
+                  date={response.updated_at}
+                  defaultValue="N/A"
+                />
+                )
+              </div>
+            ) : (
+              <Skeleton width={400} />
+            )}
+          </Text>
+          <br />
+        </div>
+      </PageSection>
+      <Tabs
+        style={{
+          marginLeft: '-18px',
+          marginRight: '-18px',
+          background: 'white',
+        }}
+        activeKey={activeTab}
+        onSelect={handleTabClick}
+      >
+        <Tab eventKey={0} title="Details">
+          <div id="test">
+            <br />
+            <Grid>
+              <GridItem offset={3} span={4}>
+                <StatusAlert
+                  status={response ? response.global_status_label : null}
+                />
+              </GridItem>
+            </Grid>
+            <br />
+            <br />
+            <Grid>
+              <GridItem span={3} rowSpan={3}>
+                <Properties hostData={response} />
+              </GridItem>
+              <GridItem style={{ marginLeft: '40px' }} span={3}>
+                <ParametersCard paramters={response.all_parameters} />
+              </GridItem>
+              <GridItem style={{ marginLeft: '40px' }} span={3} rowSpan={2}>
+                <AuditCard hostName={response.name} />
```

Comment:
When shifted down, it moves also the interfaces downwards
![Selection_337](https://user-images.githubusercontent.com/26363699/87458383-92091080-c612-11ea-9a62-cec2a58c0f9d.png)

Reply:
I use the pf4 grid system, so I assume it has some issues.
BTW I added scrolling so cards have max height, above it, a scroll bar will automatically be added